content/200-orm/500-reference/200-prisma-cli-reference.mdx

```diff
@@ -1342,6 +1342,52 @@ Other options:
    --script > script.sql
   ```
 
+## Prisma Data Platform
+
+### <inlinecode>platform</inlinecode> ([Early Access](/platform/maturity-levels#early-access))
+
+The `platform` command provides access to the Prisma Data Platform through the Prisma CLI starting in version `5.7.0` or later.
```

Comment:
```suggestion
The `platform` command provides access to the Prisma Data Platform through the Prisma CLI starting in version `5.10.0` or later.
```

Where highlighted, we should advocate for the latest version of Prisma to utilize the Platform integration in the CLI